analytics/src/analytics/cli.py

```diff
@@ -122,6 +126,29 @@ def calculate_sprint_burnup(
     )
 
 
+@export_app.command(name="test_connection")
+def test_connection() -> None:
+    """Test function that ensures the DB connection works."""
+    engine = db.get_db()
+    # connection method from sqlalchemy
+    connection = engine.connect()
+
+    # Test INSERT INTO action
+    #  splitting this to appease  E501 Line too long (184 > 100 characters will break the command
```

Comment:
For what its worth, you can split this without breaking the command, you use a multi-line string, eg.

```python
text(
    "INSERT INTO audit_log (topic,timestamp, end_timestamp, user_id, details) "
    "VALUES('test', ' 2024-06-11 10:41:15+00', '2024-06-11 10:54:15+00', 87654, 'test from command');",
)
```

Two strings next to each other creates a multi-line string.

You can also of course use no-qa on the linter.